src/pr.ts

```diff
@@ -0,0 +1,34 @@
+import { Context } from 'probot';
+import { WebhookPayloadWithRepository } from 'probot/lib/context';
+import * as semver from 'semver';
+
+export function getTargetVersions(version: string): string[] {
+  const prerelease = semver.prerelease(version);
+  const [release] = version.split('-');
+  return prerelease
+    ? prerelease[0] === 'nightly'
+      ? [`${release}-beta.1`, release]
```

Comment:
This is making the assumption the a nightly will never be released on a branch after `beta.1` which I don't think will be valid in the long run

Reply:
hm... what would you suggest as an alternative then? i'm not sure how to resolve the appropriate next version in that case. My assumption here is that `master` is the only branch which has a `-nightly` :\

Reply:
@nornagon The best thing to do is somehow depend on the `bump-version.js` script in `e/e` (I have no idea how to do that, maybe pull it in and `eval` lol) which has the full logic for calculating the next version to be deployed.